doc/release-notes.md

```diff
@@ -34,184 +34,55 @@ Core should also work on most other Unix-like systems but is not as
 frequently tested on them. It is not recommended to use Bitcoin Core on
 unsupported systems.
 
-Notable changes
-===============
+### Miniscript
 
-libbitcoinconsensus
--------------------
+- #29853 sign: don't assume we are parsing a sane TapMiniscript
 
-- libbitcoinconsensus is deprecated and will be removed for v28. This library has
-  existed for nearly 10 years with very little known uptake or impact. It has
-  become a maintenance burden.
+### RPC
 
-  The underlying functionality does not change between versions, so any users of
-  the library can continue to use the final release indefinitely, with the
-  understanding that Taproot is its final consensus update.
+- #29869 rpc, bugfix: Enforce maximum value for setmocktime
+- #29870 rpc: Reword SighashFromStr error message
 
-  In the future, libbitcoinkernel will provide a much more useful API that is
-  aware of the UTXO set, and therefore be able to fully validate transactions and
-  blocks. (#29189)
+### Index
 
-mempool.dat compatibility
--------------------------
+- #29767 Fix #29767, set m_synced = true after Commit()
```

Comment:
```suggestion
- #29776 Fix #29767, set m_synced = true after Commit()
```

Reply:
Also, this is the commit message and not the PR title but this is a bit more descriptive so makes sense.

Reply:
Thanks, addressed both.